content/influxdb/cloud/upgrade/v1-to-cloud/_index.md

```diff
@@ -0,0 +1,259 @@
+  --org your.email@example.com \
+  --token mY5uP3rS3cRe7Cl0uDt0K3n \
+  --active
+```
+
+{{% note %}}
+#### Required InfluxDB Cloud credentials
+All `influx` CLI examples below assume the required InfluxDB Cloud **host**,
+**organization**, and **authentication token** credentials are provided by your
+[`influx` CLI configuration](/influxdb/cloud/reference/cli/influx/#provide-required-authentication-credentials).
+{{% /note %}}
+
+## Create DBRP mappings
+InfluxDB database and retention policy (DBRP) mappings associate database and
+retention policy combinations with InfluxDB cloud [buckets](/influxdb/cloud/reference/glossary/#bucket).
+These mappings allow InfluxDB 1.x clients to query and write to
+InfluxDB Cloud buckets while using the 1.x DBRP convention.
+
+_For more information about DBRP mapping, see
+[Database and retention policy mapping](/influxdb/cloud/reference/api/influxdb-1x/dbrp/)._
+
+**To map a DBRP combination to an InfluxDB Cloud bucket:**
+
+1.  **Create a bucket**  
+    [Create an InfluxDB Cloud bucket](/influxdb/cloud/organizations/buckets/create-bucket/).
+    We recommend creating a bucket for each unique 1.x database and retention
+    policy combination using the following naming convention:
+
+    ```sh
+    # Naming convention
+    db-name/rp-name
+
+    # Example
+    telegraf/autogen
+    ```
+
+2.  **Create a DBRP mapping**  
+    Use the [`influx v1 dbrp create` command](/influxdb/cloud/reference/cli/influx/v1/dbrp/create/)
+    to create a DBRP mapping.
+    Provide the following:
+
+    - database name
+    - retention policy name _(not retention period)_
+    - [bucket ID](/influxdb/cloud/organizations/buckets/view-buckets/)
+    - _(optional)_ `--default` flag if you want the retention policy to be the default retention
+      policy for the specified database
+
+    {{< code-tabs-wrapper >}}
+    {{% code-tabs %}}
+[DB with one RP](#)
+[DB with multiple RPs](#)
+    {{% /code-tabs %}}
+    {{% code-tab-content %}}
+```sh
+influx v1 dbrp create \
+  --db example-db \
+  --rp example-rp \
+  --bucket-id 00xX00o0X001 \
+  --default
+```
+    {{% /code-tab-content %}}
+    {{% code-tab-content %}}
+```sh
+# Create telegraf/autogen DBRP mapping with autogen
+# as the default RP for the telegraf DB
+
+influx v1 dbrp create \
+  --db telegraf \
+  --rp autogen \
+  --bucket-id 00xX00o0X001 \
+  --default
+
+# Create telegraf/downsampled-daily DBRP mapping that
+# writes to a different bucket
+
+influx v1 dbrp create \
+  --db telegraf \
+  --rp downsampled-daily \
+  --bucket-id 00xX00o0X002
+```
+    {{% /code-tab-content %}}
+    {{< /code-tabs-wrapper >}}
+
+## Dual write to InfluxDB 1.x and InfluxDB Cloud
+Update external clients to write to your InfluxDB Cloud instance.
+**We recommend writing data to both InfluxDB 1.x and InfluxDB Cloud until you
+finish [migrating your existing time series data](#migrate-time-series-data)**.
+
+Configure external clients with your InfluxDB Cloud **host**, **organization**,
```

Comment:
```suggestion
1. Configure external clients with your InfluxDB Cloud **host**, **organization**,
```

Reply:
I didn't combine this item with the process below because this refers to more than just Telegraf. This could be client libraries, other write queues like Kafka, etc. Telegraf is just the most common, which is why I included the Telegraf update process specifically.